string message_data
---
string time_description
sensor_msgs/JointState[] trajectory
bool success
